perfkitbenchmarker/benchmark_spec.py

```diff
@@ -390,7 +392,8 @@ def PrepareVm(self, vm):
     logging.info('Waiting for boot completion.')
     for port in vm.remote_access_ports:
       vm.AllowPort(port)
-    vm.AddMetadata(benchmark=self.uid, perfkit_uuid=self.uuid)
+    vm.AddMetadata(benchmark=self.name, perfkit_uuid=self.uuid,
+                   benchmark_uid=self.uid)
```

Comment:
I'm a bit ambivalent about including benchmark_uid as metadata - whatever you prefer.